pkg/virtual/framework/rootapiserver/server.go

```diff
@@ -80,7 +80,7 @@ func NewServer(c CompletedConfig, delegationTarget genericapiserver.DelegationTa
 
 func getRootHandlerChain(c CompletedConfig, delegateAPIServer genericapiserver.DelegationTarget) func(http.Handler, *genericapiserver.Config) http.Handler {
 	return func(apiHandler http.Handler, genericConfig *genericapiserver.Config) http.Handler {
-		delegateAfterDefaultHandlerChain := genericapiserver.DefaultBuildHandlerChain(
+		delegateAfterDefaultHandlerChain := genericapiserver.DefaultBuildHandlerChainBeforeAuthz( // Question:: should this be before or after authz?
```

Comment:
double check that we don't lose any filter

Reply:
oh, this is a revert of https://github.com/sttts/kcp/pull/19/commits/0ab619e850b064057d68f1292601dbd56ccedee7 ?

Reply:
can you squash those two?